Update generate_savepoint.go

lixiangdude opened the pull request:
Drainer will encounter some errors while it starting if the savepoint filename is "savePoint"(note in there, the "P" is uppercase).

tiancaiamao commented:
LGTM

b.t.w, `b := make([]byte, n)` and `return string(b)` make two allocation, this can be further improved if you focus on the detail of optimization.

siddontang commented:
LGTM

shenli commented:
LGTM

lixiangdude committed:
Update generate_savepoint.go

Drainer will encounter some errors while it starting if the savepoint filename is "savePoint"(note in there, the "P" is uppercase).

WangXiangUSTC commented:
LGTM
